### ISSUE TITLE
Screen reader announcing incorrect row count information while navigating the table data: A11y_WinForm.NETRuntime_MonitorMrationsWindow_InfoAndRelationship

### ISSUE THREAD

**Issue by kupatkar99** [dotnet/winforms#13994]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev3; #A11yTCS; #WCAG1.3.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Info and Relationships; #Narrator; #Screen Reader; #A11yMAS;

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061

### Actual Result:
Screen reader announcing incorrect row count information while navigating the table data under “Monitor migrations” window.  

### Expected Result:
Screen reader should announce the correct row count information while navigating the table data under “Monitor migrations” window.  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle. 
2. Splash screen appears and SSMA is launched 
3. Navigate to 'Migrate Data with DMS(preview)' button and select it. 
4. 'At Scale Migration Wizard'>Enter Azure Subscriptions details 
5. Navigate all the controls under "Enter Azure subscription details" window and select next button 
6. Navigate all the controls under "source data base connection details" window and select next button  
7. Navigate all the controls under "Target database connection details" window and select next button 
8. Navigate all the controls under "select the location of the database migration" window and select "finish" button 
9. Navigate all the controls under "monitor migrations window" and select "view" button and navigate all the controls under another "monitor migration" window. 
10. Verify whether screen reader announcing correct row count or not. 

### User Empact:
When a screen reader announces incorrect row count information while navigating table data, it can mislead users about the structure and amount of content, making it harder to understand or interact with the data accurately. This affects information clarity and undermines accessibility for screen reader users.  

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-kkumar2_microsoft_com/EfK3aGoBmHxMtVS9huz7IG0B8Ixe_aVhlDx875gJGauyNA?e=EeKhEy

**Comment by merriemcgaw** [comment]
@kupatkar99 - is this applicable to .NET 10 or is this just .NET Framework? Please provide a sample project for .NET 10 and we will look into it.

**Comment by merriemcgaw** [comment]
@Olina-Zhang can your team try a quick repro with a vanilla `DataGridView`?

**Comment by Olina-Zhang** [comment]
@merriemcgaw It cannot repro in .NET 10/9 with Winforms DataGridView control, was fixed by GH PR: https://github.com/dotnet/winforms/pull/10243.

**Comment by merriemcgaw** [comment]
@kupatkar99 this is fixed in the current public release and vNext so you can close this issue please.

**Comment by merriemcgaw** [comment]
@LeafShi1 it looks like we should go ahead and service this for .NET 8. Do you mind backporting? Let @KlausLoeffelmann and me know if there was something very different that we'd need to consider.

**Comment by LeafShi1** [comment]
> [@LeafShi1](https://github.com/LeafShi1) it looks like we should go ahead and service this for .NET 8. Do you mind backporting? Let [@KlausLoeffelmann](https://github.com/KlausLoeffelmann) and me know if there was something very different that we'd need to consider.

Please review the backport PR https://github.com/dotnet/winforms/pull/14003

**Comment by kupatkar99** [comment]
As per the discussion with SME Down grading the severity to Sev3.

**Comment by merriemcgaw** [comment]
@LeafShi1 in that case we should not service this and I will close this.
